requre/helpers/files.py

```diff
@@ -46,6 +46,42 @@ class StoreFiles(ObjectStorage):
     def _test_identifier(cassette):
         return os.path.basename(cassette.storage_file)
 
+    @staticmethod
+    def store_file_content(cassette: Cassette, content: Any, file_name: str):
+        generated = f"{os.path.basename(cassette.storage_file)}.{file_name}_"
+        target_dir = os.path.realpath(os.path.dirname(cassette.storage_file))
+        # generate unique number for each file store to avoid rewrite current content
+        # increase highest number
+        suffix = f".tar.{StoreFiles.tar_compression}"
+        existing_nums = sorted(
+            [
+                int(x.replace(generated, "").replace(suffix, ""))
+                for x in os.listdir(target_dir)
+                if x.startswith(generated)
+            ]
+        )
+        next_num = 1 if not existing_nums else existing_nums[-1] + 1
```

Comment:
Is this needed when we have cassette name in the file name?

Reply:
theoretically yes, because in case you will do some operation with same directory in one test, it will lead to same file.
e.g. you will clone some repo, and then you would like to store state after commit. or something like that.

Reply:
How this differs from the situation when you are running this directly without requre? Maybe I am only not getting the reason right...